tests/test_contexts.py

```diff
@@ -54,6 +54,20 @@ def test_retrieve_contexts_via_client_using_scope_filter(self):
             contexts = self.client.contexts(scope=str(self.project.id))
             self.assertIsInstance(contexts[0], Context)
 
+    def test_retrieve_contexts_via_client_using_tags_list__contains_filter(self):
```

Comment:
For me this test fails when I run it in Pycharm. It returns a `ConnectionError`.